resources/articles/migration-service.md

```diff
@@ -0,0 +1,179 @@
+**Migration user**: The JSON object sent in the response from migration service.  
+**Migration specification**: An object used by the migration client to make requests.  
+**UserData service**: A part of the SPiD platform where the Migration client lives.
+
+## Migration service specification
+
+Your migration service:
+
+* must accept `GET` requests
+* must accept “email” query parameter with a URL encoded value
+* must respond with status code 404 if an email is not linked to a user
+* must respond with a valid UTF-8 encoded JSON object representation of the user, called MigrationUser, and status code 
+200
+* must respond within 2 seconds
+* must be highly-available
+
+## Migration specification
+
+A migration specification is a set of values that tells the migration client:
+
+* what endpoint to make the request to.
+* what header key value to include in the request to your migration service.
+               
+You will tell us what endpoint and header key value to use by using the SPiD Self-Serve page and defining a migration 
+specification for your client Id. You can define the same migration for several client Ids.
+The header key value will be included in every request to the migration service and can be used for authentication. 
+
+## The MigrationUser object
+
+Below is  a JSON schema of the MigrationUser. An example of a MigrationUser can be seen in the illustration above and 
+in the example below. 
+
+Please note that:
+
+* only the email field is required, all other fields can be omitted. 
+* invalid enum values will cause the migration to fail, while invalid locale, timezone, phone numbers and photo 
+will be removed by the migration client.
+* all times, except birthday, are in ISO8601 RFC3339 UTC format.
+* should a field not be available, please omit it from the JSON response.
+* all data should be UTF-8 encoded.
+
+```json
+{
+	"id": "urn:jsonschema:com:schibsted:spt:identity:userdataservice:models:MigrationUser",
+	"properties": {
+		"birthday": {
+			"type": "string",
+			"format": "DATE_TIME"
+		},
+		"displayName": {
+			"type": "string"
+		},
+		"sex": {
+			"type": "string",
+			"enum": [
+				"UNDISCLOSED",
+				"FEMALE",
+				"MALE"
+			]
+		},
+		"homePhone": {
+			"type": "string",
+			"description": "Format: +46761234567"
+		},
+		"fullName": {
+			"type": "string"
+		},
+		"photo": {
+			"type": "string",
+			"description": "URL"
+		},
+		"timeZone": {
+			"type": "string",
+			"description": "Any accepted Java TimeZone ID"
+		},
+		"locale": {
+			"type": "string",
+			"description": "Any accepted Java Locale ID"
+		},
+		"userId": {
+			"type": "string",
+			"description": "The id of the user in your system",
+		},
+		"mobilePhone": {
+			"type": "string",
+			"description": "Format: +46761234567"
+		},
+		"createdTime": {
+			"type": "integer",
+			"format": "UTC_MILLISEC"
+		},
+		"email": {
+			"type": "string",
+			"required": true
+		}
+	}
+}
+
+```
+
+## Repository
```

Comment:
Remove this paragraph. 99% of people reading this will not have access to our github, and therefore not the repo.